src/view/Henvendelse.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Snakkeboble from "nav-frontend-snakkeboble";
+import {HenvendelseData} from "../utils/typer";
+import { getFormattedHenvendelseDate } from "../utils/functions";
+import { Normaltekst } from 'nav-frontend-typografi';
+
+import './henvendelse.less';
+
+
+interface Props {
+    henvendelseData: HenvendelseData;
+}
+
+
+export function Henvendelse(props: Props) {
```

Comment:
Ser at det er brukt en kombinasjon av `const Komponent  = () => {}` og `function Komponent() {}` syntaks i filen.
Det er noen små forskjeller i hvordan de fungerer i javascript (hoisting etc, men det har veldig sjeldent noe å si), men jeg ville prøvd å holde til en av de to da det gir en litt renere kodebase. :)

Reply:
Klarer ikke helt å se hva du mener? 🤔 Kunne du forklart meg litt mer? For sånn jeg ser det er det bare en funksjon i filen og der har jeg brukt `function Komponent() {}`som også er brukt i de andre filene.

Reply:
I denne filen er det nok bare en funksjon ja. :) Men så ut som det var brukt en annen måte i noen andre filer. `App.tsx` ser ut til å bruke `const App: React.FC = () => {}` f.eks. 
Så tanken var bare å få dere til å ta en diskusjon rundt hvilken måte dere vil skrive komponentene deres på. :)